connectors-native-sdk-java/README.md

```diff
@@ -0,0 +1,131 @@
+# Native SDK for Connectors Java
+
```

Comment:
We should have link to docs somewhere at the top

Reply:
yup, I've defined the alias at the bottom of the doc, but forgot to use it in the text